src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+@import "../utils/variables";
+
+.header-page {
+  display: flex;
+  height: auto;
+  padding: 0 50px;
```

Comment:
To maintain consistency, use the variable `$padding-horizontal` instead of hardcoding the padding value.